### DISCUSSION TITLE
Works BUT dont WORK

### DISCUSSION THREAD

**Discussion by XxnuGGetxX-prog** [3kh0/ext-remover#80] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So when i follow the steps for ingot is shows the menu to turn the onn and off so i turn goguardian off but when i reload the page it swtches back on so i just want to know if its an update problem or what my version is Version 105.0.5195.134

**Comment by phillipbruh** [comment, marked as answer]
try using the normal one not ingot
